include/tvm/relax/block_builder.h

```diff
@@ -231,6 +232,48 @@ class BlockBuilder : public ObjectRef {
    */
   TVM_DLL static BlockBuilder Create(Optional<IRModule> ctx_mod);
 
+  /*! \brief A marker struct to disable FNormalize
+   *
+   * This struct is used as a marker to disable the use of FNormalize
+   * by this block builder.  This should only be used for TVMScript
+   * parsing, which may require producing un-normalized Relax IR for
+   * testing purposes, and to ensure that round-trips are unchanged.
+   *
+   * The name is deliberately verbose to draw attention during a code
+   * review.  The explicit default constructor prevents aggregate
```

Comment:
Great tactic :)

Reply:
Thank you.  Since any unnormalized operator is now considered ill-formed, I wanted to make it as obvious as possible when the per-operator normalization is being disabled.  It was rather amusing how many lint checks needed to be disabled in order to ensure that the parameter name occurs at every call site.  (Lint rules generally aren't aimed at making intentionally-verbose code.)